GeoMesa_Feature_List.md

```diff
@@ -0,0 +1,100 @@
+> The following document uses a hierarchical structure of features - from high level descriptions of general functionality to the subfeatures of which those general functions are composed.
+
+
+## Data Ingest/Input
+
+- geomesa-tools (command line tools for interacting with geomesa)
+  - Creating a geomesa datastore for accumulo
+    - Pointer: [CreateCommand.scala](https://github.com/locationtech/geomesa/blob/b7056fae4988ef524913bf3dc33d9ff2a3476b09/geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/accumulo/commands/CreateCommand.scala)
+    - Behavior tested; works
+  - Ingest vectors, provided a `GeoMesaInputFormat`
+    - Pointer: [IngestCommand.scala](https://github.com/locationtech/geomesa/blob/b7056fae4988ef524913bf3dc33d9ff2a3476b09/geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/accumulo/commands/IngestCommand.scala)
+    - Behavior tested; works
+  - Ingest rasters
+    - Pointer: [IngestRasterCommand.scala](https://github.com/locationtech/geomesa/blob/b7056fae4988ef524913bf3dc33d9ff2a3476b09/geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/accumulo/commands/IngestRasterCommand.scala)
+    - Supported file formats: "tif", "tiff", "geotiff", "dt0", "dt1", "dt2"
+
+
+- geomesa-convert (tools for converting various serialization formats to `SimpleFeature`s for ingestion - conversion mechanisms are specified by way of configuration files)
```

Comment:
of note, we also have predefined converters for a variety of common data formats: https://github.com/locationtech/geomesa/tree/master/geomesa-tools/conf/sfts